kitsune/wiki/utils.py

```diff
@@ -178,7 +179,42 @@ def get_featured_articles(product=None, locale=settings.WIKI_DEFAULT_LANGUAGE):
 
 
 def get_visible_document_or_404(user, **kwargs):
-    return get_object_or_404(Document.objects.visible(user, **kwargs))
+    """
```

Comment:
If you prefer to have `kwargs` here instead of default values, let's define in the docstring the standard/default args this function awaits

Reply:
I was on the fence about what to do, so if you have a preference, we can do that. On further reflection, I think default values for `look_for_translation_via_parent` and `return_parent_if_no_translation` are probably clearer.

Reply:
Agreed about the default values